src/Microsoft.ML.AutoML/Experiment/Experiment.cs

```diff
@@ -196,6 +196,22 @@ public IList<TRunDetail> Execute()
                         "was reached, and the running MLContext was stopped. Details: {0}", e.Message);
                     return iterationResults;
                 }
+                catch (AggregateException e)
+                {
+                    // This exception is thrown when the IHost/MLContext of the trainer is canceled due to
+                    // reaching maximum experiment time. Simply catch this exception and return finished
+                    // iteration results. For some trainers, Like FastTree, because training is done in parallel
+                    // in can throw multiple OperationCancelledExceptions. This causes them to be returned as an
+                    // AggregateException and misses the first catch block. This is to handle that case.
+                    if (e.InnerExceptions.All( exception => exception is OperationCanceledException))
+                    {
+                        _logger.Warning("OperationCanceledException has been caught after maximum experiment time" +
```

Comment:
(nit) pull the message out so it doesn't need to be duplicated with the above.

Reply:
Pulled out to private readonly variable.